apps/chat/src/utils/app/form-schema.ts

```diff
@@ -0,0 +1,17 @@
+import { FormButtonType } from '@/src/types/chat';
+
+import { FormSchemaButtonOption, Message } from '@epam/ai-dial-shared';
+
+export const getFormButtonType = (option: FormSchemaButtonOption) => {
+  if (option['dial:widgetOptions']?.submit) return FormButtonType.Submit;
+  return FormButtonType.Populate;
+};
+
+export const isMessageInputDisabled = (
+  messageIndex: number,
+  messages: Message[],
+) => {
+  const schema = messages[messageIndex - 1]?.custom_content?.form_schema;
```

Comment:
you have several places with `?.custom_content?.form_schema` - what about `getMessageSchema(message: Message)` method instead?